client/blocks/reader-full-post/_content.scss

```diff
@@ -442,3 +442,70 @@
 	padding-left: 16px;
 	padding-right: 16px;
 }
+
+// Writing Prompts Block Styles
+.reader-full-post__story-content .jetpack-blogging-prompt {
+	border: 1px solid var(--color-neutral-5);
+	border-radius: 2px;
+	box-sizing: border-box;
+	margin-bottom: 24px;
+	padding: 24px;
+
+	.jetpack-blogging-prompt__answers {
+		display: flex;
+
+		.image-wrapper {
+			&:first-child {
+				margin-left: -2px;
+			}
+			&:not(:first-child) {
+				margin-left: -15px;
+			}
+
+			img {
+				border: 2px solid #fff;
+				border-radius: 50%;
+				height: 24px;
+				width: 24px;
+			}
+		}
+	}
+
+	.jetpack-blogging-prompt__answers-link {
+		color: var(--studio-gray-80);
+		font-family: sans-serif;
+		font-size: $font-body;
+		margin-left: 6px;
+		text-decoration: underline;
+
+		&:hover {
+			text-decoration: none;
+		}
+	}
+
+	.jetpack-blogging-prompt__label {
+		font-family: sans-serif;
+		font-size: $font-body-small;
+		margin-bottom: 16px;
+		padding-left: 22px;
+		position: relative;
+
+		&::before {
+			background: no-repeat url(https://s0.wp.com/wp-content/mu-plugins/jetpack-plugin/production/_inc/blocks/images/icon-cd4e9ecf53fadf95246d.svg);
```

Comment:
I'm worried this url could change unexpectedly, given the hash as part of the filename. I think we should just add the icon directly to Calypso and load it from `calypso/assets/images/icons/light-bulb.svg`